backend/inspirehep/records/tasks.py

```diff
@@ -43,3 +56,49 @@ def update_records_relations(uuids):
 
     db.session.commit()
     return uuids
+
+
+@shared_task(ignore_result=False, bind=True, queue="migrator")
```

Comment:
why these random arguments? did you copy/paste this? all of them are wrong (you should read about what they do in the celery docs).